app/src/androidTest/java/nl/rug/www/rugsummerschools/controller/DrawableMatcher.java

```diff
@@ -0,0 +1,77 @@
+package nl.rug.www.rugsummerschools.controller;
+
+import android.content.res.Resources;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.drawable.Drawable;
+import android.view.View;
+import android.widget.ImageView;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+public class DrawableMatcher extends TypeSafeMatcher<View> {
+
```

Comment:
a short class comment would not be bad.

Reply:
Right, I had tried to make a matcher class in order to check a drawable that matches with id. 
I will add some comments for this though.